src/spatialdata_plot/pl/utils.py

```diff
@@ -2266,3 +2269,51 @@ def _get_transformation_matrix_for_datashader(
             tm = tm @ _get_datashader_trans_matrix_of_single_element(x)
         return tm
     return _get_datashader_trans_matrix_of_single_element(trans)
+
+
+def _datashader_shade(
```

Comment:
please pick a better, more descriptive, name. Only intuitive for people that know what datashader does